internal/compiler/typeregister.rs

```diff
@@ -129,7 +129,6 @@ pub fn reserved_properties() -> impl Iterator<Item = (&'static str, Type, Proper
         .chain(IntoIterator::into_iter([
             ("absolute-position", logical_point_type(), PropertyVisibility::Output),
             ("forward-focus", Type::ElementReference, PropertyVisibility::Constexpr),
-            ("focus", BuiltinFunction::SetFocusItem.ty(), PropertyVisibility::Public),
```

Comment:
If we remove this, there is no protection against something doing
```
property <string> focus;
```

or
```
function focus() {}
```

We need to prevent that, and there should be a test that makes sure this is not allowed

Reply:
Agreed.